debian/rules

```diff
@@ -51,6 +51,10 @@ override_dh_auto_install:
 
 override_dh_auto_build:
 
+	if ! pkg-config --modversion xfont2 1>/dev/null 2>/dev/null; then \
+	    sed -e 's/#define.*HasXfont2.*NO/#define HasXfont2 YES/' -i nx-X11/config/cf/host.def \
+	fi
```

Comment:
@Ionic: I have dropped the sedding of host.def now completely and do the auto-detection in the main Makefile. The package maintainer can then choose what Xfont API to use by making Xfont2 available (or not). Implying that "or not" means, that Xfont(1) is available instead.